docs/preview features/orchestrator.md

```diff
@@ -0,0 +1,136 @@
+# Orchestrator Preview
+Orchestrator is a powerful new language model that is optimized for conversational AI applications.
+
+- [Technical Overview](https://github.com/microsoft/botframework-sdk/blob/main/Orchestrator/docs/Overview.md)
+- [Samples](https://github.com/microsoft/BotBuilder-Samples/tree/main/experimental/orchestrator)
+
+## Orchestrator Features
+- Easy to use - Use your existing LU files from LUIS without modification
+- State of the Art - Extensive pre-trained transformer-based model for conversational applications
+- Extremely fast Inference Time: Written in C++ for performance
+- Cross-platform Support: Runs on Windows, Linux and Mac OS X, including Cloud platforms
+- Free to use: Language models are deployed alongside your bot, there are no usage fees
```

Comment:
Also runs in memory.  No calling a service (ie LUIS)

Reply:
Thanks for your input @tsuwandy, all comments addressed.